### DISCUSSION TITLE
Agregar movimientos y habilidades en JS

### DISCUSSION THREAD

**Discussion by LuisFelipeCajero** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#466] [Q&A]
Podrían ayudarme a como agregar los movimientos y habilidades del Pokédex, no me quedo muy claro si es a través de una función condicional.

**Comment by doguedogue** [comment]
Asi como insertaste la imagen (SRC), puedes hacer lo mismo con un input (VALUE) con los datos que se piden (movimientos, tipo, estadísticas, etc)

Si es con función veras que es más fácil. Ánimo tu puedes 🚀

**Reply by LuisFelipeCajero** [reply]
Muchas Gracias!
